src/radical/pilot/pmgr/launching/default.py

```diff
@@ -116,6 +116,10 @@ def finalize(self):
         except:
             self._log.exception('finalization error')
 
+        self.unregister_timed_cb(self._pilot_watcher_cb)
+        self.unregister_input(rps.PMGR_LAUNCHING_PENDING,
+                              rpc.PMGR_LAUNCHING_QUEUE, self.work)
+
```

Comment:
this block is a duplicate, the same block is set earlier inside `try...except`

Reply:
Thanks, fixed.